manifests/plugin/ansible.pp

```diff
@@ -54,6 +57,7 @@
   }
 
   include ::foreman_proxy::plugin::dynflow
+  include ::foreman_proxy::plugin::ansible::runner
```

Comment:
Do we actually need a way to turn this off? If you don't want the external repo but don't have an alternative then it'll fail.

Reply:
It will, but if you do not have alternative, than the ansible will fail as of 1.24, so I do not see a problem in failing, maybe some better warning?

Reply:
If it's a hard requirement, packaging should require it. That will have some implications so would be good to have a heads up there so we tackle it in time.

Reply:
Well it is hard requirement as of 1.24, so we can have a parameter disabling it for 1.23 and than remove it once it is hard requirement ?